api/v1alpha4/types.go

```diff
@@ -672,6 +672,10 @@ type Instance struct {
 	// Tenancy indicates if instance should run on shared or single-tenant hardware.
 	// +optional
 	Tenancy string `json:"tenancy,omitempty"`
+
+	// ID of the instance's volume
+	// +optional
+	VolumeIDs []*string `json:"volumeIDs,omitempty"`
```

Comment:
```suggestion
	VolumeIDs []string `json:"volumeIDs,omitempty"`
```
We are moving away from using pointer slices due to a conversion issue. 
https://github.com/kubernetes-sigs/cluster-api-provider-aws/pull/2495